### PR TITLE
Feature: UIDS updates

### PR DESCRIPTION
**Pull request by pyrello**
This update follows a massive update to UIDS, so every path that references that library has changed. This means that testing this requires checking as many frontend pieces as possible on sites that are using uids_base. If you need more information to get started, please ping me.

# How to test

`blt frontend` 

**Test on Home Page (https://uiowa.local.drupal.uiowa.edu)**

Test Home Page Video https://github.com/uiowa/uiowa/issues/1680
- Check "Reduce Motion" in your Accessibility settings and refresh the home page. 
Screen Shot 2020-05-29 at 3 38 32 PM

**Test home page scroll changes for (https://github.com/uiowa/uiowa/issues/1665)**
- Home page header should disappear after scrolling down the height of the header bar. 

**Test all pages on uiowa.local.drupal.uiowa.edu**
- Click through and make sure that all of the components and background images are loading correctly including the top toggle menu and secondary page menu. 
- Test form elements on https://uiowa.local.drupal.uiowa.edu/feedback

**Test on Brand Site (https://brand.local.drupal.uiowa.edu/)**

- Test lockup generator
- Test mobile/desktop header variations in (https://brand.local.drupal.uiowa.edu/admin/appearance/settings/uids_base)
- Test mobile/desktop appearance of site after parent organization checkbox is set on (https://brand.local.drupal.uiowa.edu/admin/config/system/site-information)

### CODE REVIEW

**Comment by pyrello** [comment]
I can't approve this PR, but I did test it and it looks good to me.

File: docroot/themes/custom/uids_base/gulpfile.js [hunk 1]
```diff
@@ -4,75 +4,70 @@
...
+    .pipe(postcss([ autoprefixer(), cssnano()]))
     .pipe((mode.development(sourcemaps.write('./'))))
-    .pipe(dest(config.css.dest));
+    .pipe(dest(`${paths.dest}`));
 }
 
 // Watch files.
 function watchFiles() {
-  // Watch SCSS changes.
-  watch(paths.scss + '**/*.scss', { usePolling: true }, parallel(css, copy));
```
Comment (richardbporter) [inline docroot/themes/custom/uids_base/gulpfile.js:72]: I couldn't get watch to work on Vagrant without the polling option. Did you?

Comment (pyrello) [reply docroot/themes/custom/uids_base/gulpfile.js:72]: @richardbporter I didn't test this with watch. Nice catch.

**Review by joewhitsitt — APPROVED** [review]
I only tested what was in the description on uiowa.edu and brand using blt frontend.

**Comment by pyrello** [comment]
Thanks @joewhitsitt!

**Pull request merged**